src/app/customer/trader-overview/trader-overview.component2.html

```diff
@@ -1,17 +1,20 @@
 <lk-container>
-  <!-- <div class="traders-map-row">
-    <app-map
-      class="tradersmap"
-      (locationsOnMap)="updateLocations($event)"
-    ></app-map>
-  </div> -->
   <lk-warning-box
     collapseTime="10000"
     content="Unsere Plattform befindet sich in der Testphase. Diese Testphase beschränkt
   sich aktuell auf die Geschäfte in ausgewählten Städten!"
   >
   </lk-warning-box>
-  <div class="traders-overview-row" *ngIf="hasLocations">
+  <!--<div id="map-underlay">
+    <lk-map
+      class="tradersmap"
+      (locationsOnMap)="updateLocations($event)"
+      [traderLocations]="traderLocations"
+      [position]="currentLocation"
+      (markerClicked)="traderpinClicked($event)"
+    ></lk-map>
+  </div>-->
+  <div class="traders-overview-row" id="//map-overlay" *ngIf="hasLocations">
```

Comment:
I've never seen those two slashes // before an id. Is that some convention I am not aware of?